### DISCUSSION TITLE
Some icons missing

### DISCUSSION THREAD

**Discussion by me21** [zauberzeug/nicegui#2617] [Ideas / Feature Requests]
### Description

According to https://nicegui.io/documentation/icon#material_icons_and_symbols, it should be possible to use various icon sets mentioned in [Quasar documentation](https://quasar.dev/vue-components/icon#webfont-usage). Specifically, I'd like to use themify icons. But `ui.button(icon='ti-car')` renders a button with no icon.

**Comment by falkoschindler** [comment]
Hi @me21, we currently only support material icons and symbols.

But you can simply add them with a line of custom HTML:

```py
ui.add_head_html('<link rel="stylesheet" href="https://cdn.jsdelivr.net/themify-icons/0.1.2/css/themify-icons.css" />')
ui.icon('ti-car')
```

There's probably a more recent version out there, and maybe you need to download and host the CSS and the corresponding font(s). But this is a starting point, at least.

**Reply by me21** [reply]
Thank you, it worked! I downloaded the CSS and font file, placed it into a subdirectory and wrote:
```
app.add_static_files('/static', 'static')
ui.add_head_html('<link rel="stylesheet" href="/static/css/themify-icons.css" />', shared=True)
```

**Comment by me21** [comment]
Thinking wider, it would be nice to add a paragraph to the documentation about adding custom icons and/or fonts.

**Reply by rodja** [reply]
Yes Would you like to create a pull request?

**Reply by me21** [reply]
It appears there is already an example of adding custom icons: https://nicegui.io/documentation/icon#eva_icons
Maybe just some wording change is necessary to make it obvious this approach can be used for other icon sets as well...

**Reply by me21** [reply]
Please see https://github.com/zauberzeug/nicegui/pull/2620.

**Reply by falkoschindler** [reply]
Thanks again, @me21!
The new demo has just been released in version [1.4.16](https://github.com/zauberzeug/nicegui/releases/tag/v1.4.16) and will be online shortly. 🎉

**Comment by me21** [comment]
One question: how did you find that link to `themify-icons.css` on jsDelivr? I tried searching, and I got a bunch of results, unsure how to choose the right one. Please see https://www.jsdelivr.com/?query=themify-icons.
In addition, their URLs seem to follow the pattern `cdn.jsdelivr.net/npm/{author}/{package_name}@{version}/etc/etc...`, not `cdn.jsdelivr.net/{package_name}/{version}/etc/etc...` like you shared.

**Reply by falkoschindler** [reply]
Oh, that URL was ChatGPT's suggestion. It seemed suspiciously outdated, but worked at least. 🙂
